index.php

```diff
@@ -94,14 +94,14 @@
 				</div>
 			</div>
 		</div>
-	</div>
-	<div class="column">
-		<img src="imgs/chickenWings.jpg" />
+		<div class="column">
+			<img src="imgs/chickenWings.jpg" />
+		</div>
 	</div>
 	</div> </br></br>
```

Comment:
Extra closing div on line 100
Delete the old StarBulls logo from the imgs folder

Reply:
Removed the extra div and deleted the old StarBulls logo in commit 721d614b9d67b05f9c4d20affa7dabc6267861a4